src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.UnknownOperationException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopService {
+    private final Map<String, Integer> fruitInventory = new HashMap<>();
```

Comment:
According to the encapsulation principle, the visibility of `fruitInventory` should be restricted. It would be better to make it private and provide no direct access to it from outside the class to ensure the integrity of the inventory data.